ffxivdpscalc/calc.py

```diff
@@ -78,6 +78,9 @@ def multiply_and_truncate(cls, val, factor, precision=1000):
     @classmethod
     def apply_stat(cls, damage, stat):
         return cls.multiply_and_truncate(damage, stat.get_multiplier())
+    
+    def get_gcd(self):
+        return math.floor(0.25 * (1000 - (130 * (self.speed.value - 380) // 3300))) /100
```

Comment:
i believe you can do:
`math.floor(0.25 * (1000 - self.speed.get_multiplier())) / 100`

Reply:
Done